src/app/pages/Login/Login.tsx

```diff
@@ -73,7 +73,7 @@ function Login(): JSX.Element {
           </a>
           <p>
             Don't have an account?{" "}
-            <a className={styles.signUpLink} href="#">
+            <a className={styles.signUpLink} href="./register">
```

Comment:
not sure if the . before /register is necessary here

Reply:
Thx, deleted it!